website/content/docs/v0.8/Bare Metal Platforms/equinix-metal.md

```diff
@@ -2,4 +2,109 @@
 title: "Equinix Metal"
 ---
 
-Talos is known to work on Equinix Metal; however, it is currently undocumented.
+## Prerequisites
+
+This guide assumes the user has a working API token, the Equinix Metal CLI installed, and some familiarity with the CLI.
+
+## Network Booting
+
+To install Talos to a server a working TFTP and iPXE server are needed.
+How this is done varies and is left as an exercise for the user.
+In general this requires a Talos kernel vmlinuz and initramfs.
+These assets can be downloaded from a given [release](https://github.com/talos-systems/talos/releases).
```

Comment:
should we talk here about kernel args?

required for KSPP + `talos.platform=packet`?

Reply:
👍